.yamato/all-urp_performance_boatattack.yml

```diff
@@ -33,12 +33,12 @@ URP_Performance_BoatAttack_trunk:
          rerun: on_new_revision
       -  path: .yamato/urp_performance_boatattack-win-dx12.yml#URP_Performance_BoatAttack_Win_DX12_performance_build_editmode_mono_Linear_trunk
          rerun: on_new_revision
+      -  path: .yamato/urp_performance_boatattack-osx-metal.yml#URP_Performance_BoatAttack_OSX_Metal_performance_build_editmode_mono_Linear_trunk
```

Comment:
Aren't `URP_Performance_BoatAttack_OSX_Metal_performance_playmode_runtime_mono_Linear_trunk` and `URP_Performance_BoatAttack_OSX_Metal_performance_playmode_editor_mono_Linear_trunk` supposed to be added here as well?

Reply:
explained in another comment

Reply:
I can answer this one, and sorry, I only mentioned it the [previous description](https://github.com/Unity-Technologies/Graphics/pull/6123#issue-1034937963):

> The failing pipelines are not enabled, though they resolve successfully (see fogbugz and successfully resolved above).

Those are disabled and only the in-editor job is enabled. Fixing the auth issues surfaced a crash and thus I disabled them. It is tracked [in fogbugz](https://fogbugz.unity3d.com/f/cases/1375124/). The authentication for those works as well, just don't want to introduce unstable jobs, the build job is stable though and collects metrics on build time, size and variants so it was added.